tests/ConnectionTest.php

```diff
@@ -264,6 +265,16 @@ public function testQueryLog()
 
         DB::table('items')->where('name', 'test')->delete();
         $this->assertCount(5, DB::getQueryLog());
+
+        // Error
+        try {
+            DB::table('items')->where('name', 'test')->update(
+                ['$set' => ['embed' => ['foo' => 'bar']], '$unset' => ['embed' => ['foo']]],
+            );
+            self::fail('Expected BulkWriteException');
+        } catch (BulkWriteException) {
+            $this->assertCount(6, $logs = DB::getQueryLog());
```

Comment:
`$logs` isn't used anywhere
```suggestion
            $this->assertCount(6, DB::getQueryLog());
```